smoketest.sh

```diff
@@ -102,12 +102,26 @@ function runDemoApps() {
         --name quarkus-test-plugin \
         --pod cryostat-pod \
         --restart unless-stopped \
+        --env QUARKUS_HTTP_PORT=10010 \
         --env ORG_ACME_CRYOSTATSERVICE_AUTHORIZATION="Basic $(echo -n user:pass | base64)" \
         --env ORG_ACME_CRYOSTATSERVICE_MP_REST_URL="${protocol}://cryostat:${webPort}" \
         --env ORG_ACME_CRYOSTATSERVICE_CALLBACK_HOST="cryostat" \
         --env ORG_ACME_JMXHOST="cryostat" \
         --env ORG_ACME_JMXPORT="9097" \
-        -d quay.io/andrewazores/quarkus-test:0.0.6
+        -d quay.io/andrewazores/quarkus-test:0.0.8
+
+    podman run \
+        --name quarkus-test-plugin-2 \
+        --pod cryostat-pod \
+        --restart unless-stopped \
+        --env QUARKUS_HTTP_PORT=10011 \
+        --env JAVA_OPTIONS="-Dquarkus.http.host=0.0.0.0 -Djava.util.logging.manager=org.jboss.logmanager.LogManager -Dcom.sun.management.jmxremote.port=9197 -Dcom.sun.management.jmxremote.ssl=false -Dcom.sun.management.jmxremote.authenticate=false" \
```

Comment:
What's the difference between these `quarkus-test-plugin-2`? This one has extra java options args.

Also there are 2 plugin in the same port in the comments.
```
    # 9096: quarkus-test RJMX
    # 9096: quarkus-test-plugin RJMX
```

Reply:
`quarkus-test-plugin-2` is running the same code as `quarkus-test-plugin`, but the `JAVA_OPTIONS` run it with a different JMX port than the container's default (9197 vs the default of 9097 actually, that comment is outdated).

This is just to test the case of two identical plugins registering themselves with the same realm name.

Reply:
The defaults are here btw:

https://github.com/andrewazores/quarkus-test/blob/e91afb589e961f157c472e24e57b5977ed73cf1b/src/main/docker/Dockerfile.jvm#L44

Reply:
Ah, I see thanks!